### PR TITLE
fix(ps): Don't show scanning screen on accept after review

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview
Accepting a ballot after review only takes about half a second. Previously, we showed the "Please wait, checking your ballot" screen, since it used the same code path as a ballot that didn't need review.

To fix this, we add a new state `accepting_after_review` to the state machine (which uses the same implementation as `accepting` - finally figured out how to reuse state machine logic).

The frontend is updated to disable the buttons in the modal for that half second, which provides enough visual feedback that the button press actually did something without being too disruptive.

## Demo Video or Screenshot

https://user-images.githubusercontent.com/530106/184453020-343fc88f-8f17-4e0f-9b79-036eefb6d82c.mov

## Testing Plan 
- Updated existing tests
- Manually tested with ballots of each review type

## Checklist
- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [ ] I have added JSDoc comments to any newly introduced exports

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: services/scan/src/precinct_scanner_state_machine.ts [hunk 1]
```diff
@@ -117,11 +119,11 @@ function connectToPlustek(createPlustekClient: CreatePlustekClient) {
   };
 }
 
-async function disconnectFromPlustek({ client }: Context) {
+async function closePlustekClient({ client }: Context) {
```
Comment (jonahkagan) [inline services/scan/src/precinct_scanner_state_machine.ts:122]: unrelated name change

File: frontends/precinct-scanner/src/screens/scan_warning_screen.tsx [hunk 2]
```diff
@@ -197,20 +222,13 @@ function UndervoteWarningScreen({
                   </span>
                 )}
               </p>
-              <Text italic>
+              <Text italic small>
```
Comment (carolinemodic) [inline frontends/precinct-scanner/src/screens/scan_warning_screen.tsx:225]: why are we also making the text smaller?

**Review by benadida — APPROVED** [review]

**Review by carolinemodic — APPROVED** [review]

File: frontends/precinct-scanner/src/screens/scan_warning_screen.tsx (hunk 2 above)
Comment (jonahkagan) [reply frontends/precinct-scanner/src/screens/scan_warning_screen.tsx:225]: It better matches the hierarchy of text on the screen before opening the modal
